primo-zip.py

```diff
@@ -61,14 +59,14 @@ def main():
 
     # find each file in dir structure, replace tags with secrets
     swap_secrets_and_values(secret_data, types_to_check,
-                            directory, secret_visible=True)
+                            directory, True)
```

Comment:
@ztucker4 The variable names should remain in these `swap_secrets_and_values()` calls, so we know what the `True` and `False` represent - so, restore `secret_visible=True` etc. here.  It was just the default value in the method declaration which needed removing, which you did.